runtime/tests/org.eclipse.core.tests.harness/META-INF/MANIFEST.MF

```diff
@@ -2,11 +2,10 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Eclipse Core Tests Harness
 Bundle-SymbolicName: org.eclipse.core.tests.harness;singleton:=true
-Bundle-Version: 3.12.300.qualifier
+Bundle-Version: 3.13.0.qualifier
 Bundle-Vendor: Eclipse.org
-Export-Package: org.eclipse.core.tests.harness,
- org.eclipse.core.tests.session,
- org.eclipse.core.tests.session.samples
+Export-Package: org.eclipse.core.tests.harness;version="3.12.300",
```

Comment:
Just keep in mind that there is no need to have equal version on bundle+package, so you can start with 1.0

Reply:
Done. And used "1.1" for the package with new API. Just to distinguish. 
As I plan to remove the unused fields from `ConfigurationSessionTestSuite` the package version has to be bumped to 2.0 then.